### ISSUE TITLE
Provide plain text inline version: OPSEC as well as less web requests on the wire

### ISSUE THREAD

**Issue by marsrobertson** [TechForPalestine/palestine-banner#4]
Small file: https://tfpb.techforpalestine.org/lib/banner.min.js

Small file: https://tfpb.techforpalestine.org/lib/banner.min.css

I did similar project 10 years ago: https://github.com/genesisdotre/cmdctrl

**UPSIDE:**
* everything inline
* less web requests
* better for security

**DOWNSIDE:**
* unable to update later on

**Comment by smashah** [comment]
Yeah i think there should be options to have all different variations be one liners so it's just a drop-in thing
